src/index.js

```diff
@@ -1,37 +1,37 @@
 import './styles/styles.css';
-import getData, { postDataLikes, getLikesData } from './module/data.js';
+
+import getData, { postDataLikes, getLikesData, countItem } from './module/data.js';
 import showReservations from './module/reservation.js';
 import { createCommentPop, displayComments } from './module/comment_display_generation.js';
 import { getMeals, postComment } from './module/comment_api_functions.js';
 import { commentAddSuccess, commentAddError } from './module/comment_response_messages.js';
 
 const card = document.querySelector('.list-items .card');
-getData().then((data) => {
-  data.meals.forEach((item) => {
-    card.innerHTML += `
-    <li id="${item.idMeal}">
-          <div class="card-image">
-          <p class = "hideMe">${item.idMeal}</p>
-            <img src="${item.strMealThumb}" alt="${item.strMeal}">
+const data = await getData();
```

Comment:
Than why  I told you to move all function to a specific module and then you can call then

Reply:
I'm telling you again n again, my reservation only exists within this block. If i were to do that , it will take me 12 hours and since time is of essence all you can do is just put reservation.js in .then() block and it will work .

Reply:
No need of taking 12 hours we did pretty much the same and you can even have a look at it.... now you create a conflict with the base code Can you please try to look around and maybe you can perform it better... Thanks

Reply:
My brother, Everything is working fine, we are behind schedule. All you need to do is just put back reservation where it was thats it. Nothing more then that .

Everything is implemented correctly and working fine with current development branch .

Please just put back the code you deleted.

Reply:
What does it break exactly bro @shuja-shah ? my code is not related to yours

Reply:
Do you see that I worked with the likes or even @PraisesPJMT worked with the comment, You could do the same to set the item display free any time the reservation may break to not break the production displaying on the home page
See the way I worked with like it below code:

```
span.forEach((sp) => likes.filter((like) => {
  if (sp.parentNode.parentNode.id === like.item_id) {
    sp.innerHTML = `
      <strong>${like.likes}</strong> Likes
    `;
  }
  return 0;
}));
```